### PR TITLE
User Interface Implementation

### PR DESCRIPTION
**Pull request by Kevin-Guillen**
Capture

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,113 +1,18 @@
...
-      ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
-    );
-  }
-}
+void main() => runApp(MyApp());
```
Comment (MarcosCGlobant) [inline lib/main.dart]: Missed extra line here, do Ctrl + Alt + L to fix it

**Review by MarcosCGlobant — APPROVED** [review]
Just one minor comment, LGTM

File: lib/main.dart (hunk 1 above)
Comment (Kevin-Guillen) [reply lib/main.dart]: Fixed

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**